qiita_db/study.py

```diff
@@ -921,7 +957,46 @@ def ebi_submission_status(self, value):
 
     ebi_submission_status.__doc__.format(', '.join(_VALID_EBI_STATUS))
 
-    # --- methods ---
+    @property
+    def tags(self):
+        """Returns the tags of the study
+
+        Returns
+        -------
+        list of str
+            The study tags
+        """
+        with qdb.sql_connection.TRN:
+            sql = """SELECT study_tag_id, study_tag
+                        FROM qiita.study_tags
+                        LEFT JOIN qiita.per_study_tags USING (study_tag_id)
+                        WHERE study_id = {0}""".format(self._id)
+            qdb.sql_connection.TRN.add(sql)
+            return qdb.sql_connection.TRN.execute_fetchindex()
+
+    @tags.setter
+    def tags(self, tag_ids):
```

Comment:
This is not setting the tags correctly. If a tag has been removed from the list it doesn't get removed from the DB.

Would it be more useful to add a couple of functions `add_tags` and `remove_tags`?